src/core/Object3D.js

```diff
@@ -466,6 +466,28 @@ class Object3D extends EventDispatcher {
 
 	}
 
+	getAllObjectsByProperty( name, value ) {
```

Comment:
The projects (at least in the core) does not use the  `All*` nomenclature. Similar to `Raycaster`'s methods `intersectObject()` and `intersectObjects()`, I would name the method `getObjectsByProperty()`

Reply:
I named it that way before, but thought that for users/devs it would be easier to notice the 'All' versus the added 's', and added it like it's done in Unity for example.

Should i change it back anyway? i understand that it's more consistent

Reply:
Yes, please revert.

Reply:
Done